tardis/plasma/tests/test_complete_plasmas.py

```diff
@@ -1 +1,122 @@
+import os
 
+import pandas as pd
+import pytest
+from pandas.util import testing as pdt
+from numpy.testing import assert_almost_equal
+from tardis.io.config_reader import Configuration
+from tardis.simulation import Simulation
+
+
+class BasePlasmaTest(object):
+    #Class defining all common tests for different setups of Plasma
+    #This can then be inherited for different Plasma setup
+
+    @pytest.fixture(scope="class", autouse=True)
+    def plasma(self, atomic_data_fname, tardis_ref_path, reference_file_path, config_path):
+        self.config = Configuration.from_yaml(config_path)
+        self.config['atom_data'] = atomic_data_fname
+        self.sim = Simulation.from_config(self.config)
```

Comment:
If I read the tests correctly, there is no need to set variables on `self` as they are never checked.